UnitTests/BraintreeShopperInsightsTests/BTShopperInsightsClient_Tests.swift

```diff
@@ -194,6 +206,13 @@ class BTShopperInsightsClient_Tests: XCTestCase {
         XCTAssertEqual(mockAPIClient.postedAnalyticsEvents.first, "shopper-insights:paypal-presented")
     }
     
+    func testSendPayPalPresentedEvent_whenPaymentMethodsDisplayedNotNil_sendsAnalytic() {
+        let paymentMethods = ["Apple Pay", "Card", "PayPal"]
+        sut.sendPayPalPresentedEvent(paymentMethodsDisplayed: paymentMethods)
+        XCTAssertEqual(mockAPIClient.postedPaymentMethodsDisplayed, paymentMethods.joined(separator: ","))
```

Comment:
I think this test will fail because it expects a space after the comma:
```suggestion
        XCTAssertEqual(mockAPIClient.postedPaymentMethodsDisplayed, paymentMethods.joined(separator: ", "))
```